cdap-api-common/src/main/java/co/cask/cdap/api/data/schema/Schema.java

```diff
@@ -732,22 +729,27 @@ private boolean checkCompatible(Schema target, Multimap<String, String> recordCo
   }
 
   /**
-   * Creates a map of indexes based on the iteration order of the given set.
+   * Creates a pair of Maps such that the first map contains indexes based on the iteration order of the given set.
+   * The second map is the reverse of the first map.
```

Comment:
`BiMap` class uses the method named `inverse` for this operation also apache commons uses `MapUtils.invertMap`, so thought reverse may not be correct usage. However since this is a private method not a big deal :+1: